framework/src/main/java/org/checkerframework/framework/util/typeinference8/constraint/ConstraintSet.java

```diff
@@ -162,26 +184,32 @@ public void remove(ConstraintSet subset) {
    */
   public ConstraintSet getClosedSubset(Dependencies dependencies) {
     ConstraintSet subset = new ConstraintSet();
-    Set<Variable> inputDependencies = new LinkedHashSet<>();
-    Set<Variable> outDependencies = new LinkedHashSet<>();
+    Set<Variable> allOutputsOfC = new LinkedHashSet<>();
```

Comment:
What is "C"?  Is it the variable defined [below](https://github.com/typetools/checker-framework/pull/7002/files#diff-443aa8ad97aad7a9e92c2f377eee7e690b5915c2df3ead07cde153dfd80c34dfR198)?
How will this be used, or why do we care about it?